finnishtransportagency/vaylariski-frontend
# ANALPA-3365 fix container vulnerabilities

## Jira issue



## Description



Fix container vulnerabilities by doing the following changes in dockerfiles (both local Dockerfile and Dockerfile.aws):
- upgrade baseimage
- add `apk upgrade` statement
- pushed the new baseimage to our aws ecr so we can pull our mirrored version in Dockerfile.aws (as we did before with the previous baseimage version)

### How to test
- install `trivy`: https://github.com/aquasecurity/trivy
- (you can build and scan the dev branch's container here for comparison)
- checkout this branch
- build the frontend container
- scan the frontend container with `trivy image vaylariski-frontend` (you name may vary)
- the scan should reveal little to no vulnerabilities, no critical

## What type of PR is this? (check all applicable)


## Added tests?


## Added to documentation?

- [ ] 👨🏻‍💻 documented in code: docstrings and typing

## Checklist


- [x] I have followed good coding conventions, eg [these](https://peps.python.org/pep-0008/)
- [ ] The code has been formatted using **isort .**, **pre-commit run --all-files**
- [ ] Tests have been run and they pass
- [ ] New or updated dependencies have been added to `requirements.txt` and/or `requirements_dev.txt`
